libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -130,15 +128,16 @@ void GCS::enable_high_latency_connections(bool enabled)
  */
 bool GCS::install_alternative_protocol(mavlink_channel_t c, GCS_MAVLINK::protocol_handler_fn_t handler)
 {
-    if (c >= num_gcs()) {
+    auto *link = chan(c);
```

Comment:
"link" is pretty generic.  how about "mav_chan" instead?

Reply:
Ah, I hadn't responded to this.  I was trying to avoid the word "channel" here as that's the *guts* of the mavlink library.  It essentially nominates a buffer in the mavlink library to be used for processing inbound mesages, and not much more.

In this case "link" is an instance of "GCS_MAVLink" - it contains all of ArduPilot's state about that link to the GCS.